packages/flutter_tools/lib/src/linux/linux_doctor.dart

```diff
@@ -170,6 +170,9 @@ class LinuxDoctorValidator extends DoctorValidator {
     } on ArgumentError {
       // ignore error.
     }
+    on ProcessException {
```

Comment:
nit this should be on the previous line
```suggestion
    on ProcessException {
```